src/igvfd/schemas/construct_library_set.json

```diff
@@ -756,10 +756,12 @@
             "description": "The type of control this file set represents.",
             "type": "string",
             "enum": [
-                "control transduction"
+                "control transduction",
+                "non-targeting guides"
             ],
             "enum_descriptions": {
-                "control transduction": "This construct is tranduced in the sample to serve as a control."
+                "control transduction": "This construct is tranduced in the sample to serve as a control.",
```

Comment:
Since this line is getting edited anyway, could we fix the typo? "tranduced"

Reply:
fixed